source/fonts/README.md

```diff
@@ -0,0 +1,6 @@
+Font used in a `@font-face` declaration in will be inlined or copied to
```

Comment:
Should this be 'Fonts used in a @font-face declaration will be inlined or copied to` ?